depthai_demo.py

```diff
@@ -244,6 +244,13 @@ def startLoop(self):
                     #mjpeg = packetData
                     #mat = cv2.imdecode(mjpeg, cv2.IMREAD_COLOR)
                     #cv2.imshow('mjpeg', mat)
+                elif packet.stream_name == 'color':
+                    meta = packet.getMetadata()
+                    w = meta.getFrameWidth()
+                    h = meta.getFrameHeight()
+                    yuv420p = packetData.reshape( (h * 3 // 2, w) )
+                    bgr = cv2.cvtColor(yuv420p, cv2.COLOR_YUV2BGR_IYUV)
+                    cv2.imshow("color", bgr)
```

Comment:
add FPS counter to the image frame

Reply:
Will issue a fix directly in `develop` branch

Reply:
Added in: 132295df28031a2615640ce15bcf1b983a80692f